frontend/src/assets/content/shoshone/rules-to-know/cutting-your-tree.md

```diff
@@ -1,12 +1,12 @@
-[comment]: <> (TREEHEIGHT, STUMPHEIGHT and STUMPDIAMETER are replaced with the values in the database.)
+[comment]: <> ({{treeHeight}}, STUMPHEIGHT and {{stumpDiameter}} are replaced with the values in the database.)
```

Comment:
Forgot to update STUMPHEIGHT here @sadlerw

Reply:
kk